lib/ffmpeg_cli.dart

```diff
@@ -9,7 +9,6 @@ export 'src/ffprobe/ffprobe_json.dart';
 export 'src/ffmpeg/ffmpeg_command.dart';
 export 'src/ffmpeg/ffmpeg_command_builder.dart';
 export 'src/ffmpeg/log_level.dart';
-export 'src/ffmpeg/video_size.dart';
```

Comment:
This line still needs to come back so that projects can use the video size

Reply:
Oh my bad, didn't realise this got removed.